esphome/components/i2s_audio/media_player/__init__.py

```diff
@@ -69,6 +70,7 @@ def validate_esp32_variant(config):
                     cv.Optional(CONF_MODE, default="mono"): cv.one_of(
                         *EXTERNAL_DAC_OPTIONS, lower=True
                     ),
+                    cv.Optional(CONF_I2S_COMM_FMT, default=False): cv.boolean,
```

Comment:
Reading through the function in the audio library, the boolean represents LSB or MSB, so either the key should change here, or it should be an enum value between `LSB` and `MSB`

```suggestion
                    cv.Optional(CONF_I2S_COMM_FMT, default=False): cv.boolean,
```

Reply:
Sure. I've added the enum values to the yaml file, so between `lsb` and `msb` (being `msb` the default value to avoid any breaking changes)